scripts/publish_doxygen_coverage.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+const jwt = require('jsonwebtoken');
+const github = require('@octokit/rest')();
+const zlib = require('zlib');
+const AWS = require('aws-sdk');
+const fs = require('fs');
+
+const SIZE_CHECK_APP_ID = 14028;
```

Comment:
Can we use same APP ids? I think same are used for binary size scripts.

Reply:
I don't think it makes sense to create a new APP id for each metric we collect. Perhaps we could rename it?

Reply:
I would be fine renaming it so that we don't make an app ID for every metric. I think this requires a GitHub account in the Mapbox org with admin privileges though as I'm unable to see the Size Checker GH App here.